pkgs/tools/admin/clair/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildGoPackage, fetchFromGitHub, makeWrapper, rpm, xz }:
+
+buildGoPackage rec {
+  pname = "clair";
+  version = "2.0.7";
+
+  goPackagePath = "github.com/coreos/clair";
+
+  src = fetchFromGitHub {
+    owner = "coreos";
+    repo = "clair";
+    rev = "v${version}";
+    sha256 = "0n4pxdw71hd1rxzgf422fvycpjkrxxnvcidys0hpjy7gs88zjz5x";
+  };
+
+  buildInputs = [ makeWrapper ];
```

Comment:
The `makeWrapper` dependecy should be in `nativeBuildInputs`. The purpose is explained in the [NixOS manual](https://nixos.org/nixpkgs/manual/#ssec-stdenv-dependencies).

Reply:
Fixed, thanks!